src/extensions/walls/WallsApplicationCustomizer.ts

```diff
@@ -56,12 +71,12 @@ export default class WallsApplicationCustomizer
     // Site contents page
     if (this.context.pageContext.site.serverRequestPath === "/_layouts/15/viewlsts.aspx") {
       window.setTimeout(() => {
-        let commandBar = document.querySelector(".ms-CommandBar-secondaryCommand");
+        let commandbar = document.querySelector(".ms-commandbar-secondarycommand");
```

Comment:
After this change here, the Site workflows and Site settings buttons in the command bar will remain on the Site Contents page.